package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1098,7 +1098,7 @@ def omega_selections(residues, c_name='C', n_name='N', ca_name='CA'):
     transplants[ResidueGroup].append(('omega_selections', omega_selections))
 
     def chi1_selection(residue, n_name='N', ca_name='CA', cb_name='CB',
-                       cg_name='CG'):
+                       cg_name='*G *G1 and not name H*'):
```

Comment:
Feels a bit hacky – can't we be more explicit and say CG CG1 CG2 ... or whatever the official definition might be? Ultimately this will only work for standard residues anyway (at least in default) so as long as it works for typical PDB structures and MD structures closely following PDB standards, I think it will be ok.

Reply:
I did go around hunting for a chi1 definition, since I always understood it to be to whichever heavy atom was hanging off C-beta (so therefore ill-defined for branched aa's). Both this [EBI practical](https://www.ebi.ac.uk/pdbe/modval5) and [garlic](http://www.mlb.co.jp/linux/science/garlic/doc/commands/dihedrals.html) specify CG, CG1, OG1 or SG for the canonical amino acids -- no CG2. We can be explicit, yes. I leaned towards flexibility because of that recent question with the custom DNA residue and people using more and more ncAAs. However the arguments in the method are so easily modified that maybe the defaults should be more strictly sensible.